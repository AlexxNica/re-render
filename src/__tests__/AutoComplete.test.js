import React from 'react';
import renderer from 'react-test-renderer';
import AutoComplete from '../AutoComplete';

it('renders correctly', () => {
    const tree = renderer.create(
        <AutoComplete list={['Apple', 'Mango']}
                      ignoreCase={true}
                      onItemSelected={(item) => {
                      }}
                      placeholder={'Search fruit'}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
