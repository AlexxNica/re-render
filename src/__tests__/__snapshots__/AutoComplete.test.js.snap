// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="rn-alignItems-1oszu61 rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-6koalj rn-flexShrink-1qe8dj5 rn-flexBasis-1mlwlqe rn-flexDirection-eqz5dr rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-position-bnwqim"
  style={
    Object {
      "width": "200px",
    }
  }
>
  <input
    autoCapitalize="sentences"
    autoComplete="on"
    autoCorrect="on"
    className="rn-appearance-30o5oe rn-backgroundColor-wib322 rn-borderTopColor-kqr9px rn-borderRightColor-q0dj5p rn-borderBottomColor-1ah7hsa rn-borderLeftColor-137uh4u rn-borderTopLeftRadius-ou6ah9 rn-borderTopRightRadius-t12b5v rn-borderBottomRightRadius-zmljjp rn-borderBottomLeftRadius-pm2fo rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-color-homxoj rn-font-1lw9tu2 rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-resize-1dz5y72"
    dir="auto"
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    onKeyPress={[Function]}
    onSelect={[Function]}
    placeholder="Search fruit"
    readOnly={false}
    style={
      Object {
        "height": "40px",
        "width": "200px",
      }
    }
    type="text"
    value=""
  />
</div>
`;
